DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/shannonization/src/IntersectionKernel.hpp

```diff
@@ -147,10 +147,10 @@ struct IntersectionKernel<2, II, APipe, BPipe> {
     // initialize variables
     unsigned int a = APipe::read();
     unsigned int b = BPipe::read();
-    int a_count = 1;
-    int b_count = 1;
-    int a_count_next = 2;
-    int b_count_next = 2;
+    //int a_count = 1;
+    //int b_count = 1;
+    //int a_count_next = 2;
+    //int b_count_next = 2;
```

Comment:
Why are we leaving dead code here rather than deleting these lines?  If these are meant to illustrate something, perhaps you should add a comment saying why these lines are commented out and what they are meant to illustrate?

Comment applies to other instances of commented out code in this file

Reply:
Addressed. tested in emulation and reports.